azkaban-common/src/test/resources/azkaban/executor/embedded/innerFlow.job

```diff
@@ -0,0 +1,5 @@
+type=javaprocess
+java.class=azkaban.test.executor.SleepJavaJob
```

Comment:
I am not sure if you will consider it as hack. Can you have a look at https://github.com/azkaban/azkaban/blob/multipleexecutors_canary/azkaban-common/src/test/java/azkaban/utils/TestUtils.java approach ?